<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProductWidget</class>
 <widget class="QWidget" name="Produtos">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Cadastro de Produtos</string>
  </property>
  <property name="styleSheet">
   <string notr="true">
    QWidget {
        background-color: #333;
    }
    QLabel {
        color: white;
        margin: 5px;
        font-family: "Arial Black";
    }
    QLineEdit, QDateEdit {
        background-color: #3C3C39;
        color: white;
        border: 1px solid #555;
        border-radius: 5px;
        padding: 1px 5px;
        margin: 5px;
        height: 25px;
        font-family: "Arial Black";
    }
    QPushButton {
        background-color: #5C5C59;
        color: white;
        border-radius: 5px;
        padding: 5px;
        margin: 10px;
        height: 40px; /* Altura fixa em pixels */
        font-family: "Arial Black";
    }
    QPushButton:hover {
        background-color: #6C6C69;
    }
    QComboBox {
        background-color: #3C3C39;
        color: white;
        border: 1px solid #555;
        border-radius: 5px;
        padding: 1px 5px;
        margin: 5px;
        height: 25px;
        font-family: "Arial Black";
    }
    QTableWidget {
        background-color: #3C3C39;
        color: white;
        border: 1px solid #555;
        border-radius: 5px;
        font-family: "Arial Black";
    }
    QTableWidget QHeaderView::section {
        background-color: #5C5C59;
        color: white;
        font-family: "Arial Black";
    }
    QTableWidget QHeaderView::section:checked {
        background-color: #6C6C69;
    }
    QTableWidget::item {
        background-color: #3C3C39;
        color: white;
    }
   </string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>10</number>
   </property>
   <property name="margin" stdset="0">
    <number>10</number>
   </property>
   <item>
    <widget class="QWidget" name="formContainer" native="true">
     <layout class="QVBoxLayout" name="verticalLayoutContainer">
      <property name="spacing">
       <number>10</number>
      </property>
      <property name="margin" stdset="0">
       <number>10</number>
      </property>
      <item alignment="Qt::AlignHCenter">
       <widget class="QLabel" name="label">
        <property name="font">
         <font>
          <family>Arial Black</family>
          <pointsize>36</pointsize>
          <bold>true</bold>
         </font>
        </property>
        <property name="text">
         <string>Cadastro de Produtos</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QGridLayout" name="formLayout">
        <property name="spacing">
         <number>15</number>
        </property>
        <item row="0" column="0">
         <widget class="QLabel" name="labelName">
          <property name="text">
           <string>Nome do Produto:</string>
          </property>
         </widget>
        </item>
        <item row="0" column="1" colspan="2">
         <widget class="QLineEdit" name="editName"/>
        </item>
        <item row="1" column="0">
         <widget class="QLabel" name="labelPrice">
          <property name="text">
           <string>Preço por Unidade:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="editPrice"/>
        </item>
        <item row="1" column="2">
         <widget class="QLabel" name="LabelQuantity">
          <property name="text">
           <string>Quantidade Adicionada no Estoque:</string>
          </property>
         </widget>
        </item>
        <item row="1" column="3">
         <widget class="QLineEdit" name="editQuantity"/>
        </item>
        <item row="2" column="0">
         <widget class="QLabel" name="labelSupplier">
          <property name="text">
           <string>Nome do Fornecedor:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="1">
         <widget class="QLineEdit" name="editSupplier"/>
        </item>
        <item row="2" column="2">
         <widget class="QLabel" name="labelDate">
          <property name="text">
           <string>Data da Adição:</string>
          </property>
         </widget>
        </item>
        <item row="2" column="3">
         <widget class="QDateEdit" name="dateEdit"/>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="buttonLayout">
        <property name="spacing">
         <number>10</number>
        </property>
        <item>
         <widget class="QPushButton" name="buttonSubmit">
          <property name="text">
           <string>Cadastrar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonUpdate">
          <property name="text">
           <string>Alterar</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QPushButton" name="buttonDelete">
          <property name="text">
           <string>Deletar</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QTableWidget" name="tableWidget">
     <column>
      <property name="text">
       <string>Id</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Nome</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Quantidade</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Preço</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Fornecedor</string>
      </property>
     </column>
     <column>
      <property name="text">
       <string>Data</string>
      </property>
     </column>
    </widget>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
